data/system_actions.ron

```diff
@@ -5,6 +5,8 @@
     BrightnessDown: "busctl --user call com.system76.CosmicSettingsDaemon /com/system76/CosmicSettingsDaemon com.system76.CosmicSettingsDaemon DecreaseDisplayBrightness",
     /// Increases screen brightness
     BrightnessUp: "busctl --user call com.system76.CosmicSettingsDaemon /com/system76/CosmicSettingsDaemon com.system76.CosmicSettingsDaemon IncreaseDisplayBrightness",
+    /// Switch between input sources
+    InputSourceSwitch: "cosmic-settings keyboard input-source switch",
```

Comment:
Hmm this feels like something that would be better as a dbus service in e.g. cosmic-settings-daemon (especially with regards to compatibility in other compositors), but I guess this works for now.

Reply:
I can definitely move it